chore(readme): make Piped less prominent in our README (#8429)
libre-tube/LibreTube: README.md

YouTube has an extremely invasive [privacy policy](https://support.google.com/youtube/answer/10364219) which relies on using user data in unethical ways. They store a lot of your personal data - ranging from ideas, music taste, content, political opinions, and much more than you think.

> LibreTube aims at improving the users' privacy by being independent from Google and bypassing their data collection as much as possible. The app only sends the minimum amount of data necessary to ensure that the app works, e.g. it only makes one request to YouTube to load the video you want to play instead of sending additional requests about your behavior while using the app.

Reviewer: ```suggestion
LibreTube aims at improving the users' privacy by being independent from Google and bypassing their data collection as much as possible. The app only sends the minimum amount of data necessary to ensure that the app works, e.g. it only loads the YouTube-video, without transmitting your behavior when using the app.
```

I wouldn't put so much focus on the requests (since it's a technical term, not every user might understand). Also, it dependents how many requests are necessary to load a video :)

Differences to NewPipe
</h2>

> LibreTube's main difference to NewPipe is its design: LibreTube uses [Material 3 Expressive](https://m3.material.io), while NewPipe uses [Material Design 1](https://m1.material.io/) (which was initially released in 2014). Additionally, LibreTube has some advanced features that NewPipe doesn't, e.g. SponsorBlock support or account synchronization via Piped. <!-- TODO: replace with LibreTube sync server -->

Reviewer: I would argue that our entire project has a stronger focus on the user experience, with a more modern design being just on part of that. Others being are the best-of-both worlds approach for the subscription feed, the upcoming sync server, built-in SponsorBlock, RYD, DeArrow, etc.

While LibreTube only supports content from YouTube, NewPipe also allows the use of other platforms like SoundCloud, PeerTube, Bandcamp and media.ccc.de.

> Both, LibreTube and NewPipe, are great clients for watching YouTube videos. There's no general answer about which one is better, just try them both and see which one fits you best.

Reviewer: That's just false, LibreTube is obviously better :)
